src/resources/js/tickets.js

```diff
@@ -329,6 +325,12 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 
 			// then show the one we want
 			$panel.attr( 'aria-hidden', false );
+
+			if ( ! $panel.is( $base_panel ) ) {
+				$( window ).on( 'beforeunload', beforeUnload );
+			} else {
+				$( window ).off( 'beforeunload' );
```

Comment:
You should add the handler here so you're not removing _allthethings_?(there may be third-party handlers)